server/schedulers/utils_test.go

```diff
@@ -50,3 +51,80 @@ func (s *testMinMaxSuite) TestMinDuration(c *C) {
 	c.Assert(minDuration(time.Second, time.Minute), Equals, time.Second)
 	c.Assert(minDuration(time.Second, time.Second), Equals, time.Second)
 }
+
+var _ = Suite(&testScoreInfosSuite{})
+
+type testScoreInfosSuite struct{}
+
+func (s *testScoreInfosSuite) TestSortScoreInfos(c *C) {
+	{
+		scoreInfos := NewScoreInfos()
+		expectedScores := []float64{0.2, 0.4, 0.4, 0.4}
+
+		for i, score := range expectedScores {
+			scoreInfos.Add(NewScoreInfo(uint64(i+1), score))
+			c.Assert(scoreInfos.isSorted, IsFalse)
+		}
+
+		c.Assert(scoreInfos.Min().score, Equals, 0.2)
```

Comment:
This test is weak. `0.2` is at `scoreInfos[0]` even if `Sort()` is not called. Consider generate some random data to  test, instead of simple manual data.